pkg/svcat/service-catalog/plan.go

```diff
@@ -155,3 +185,42 @@ func (sdk *SDK) RetrievePlanByClassAndPlanNames(className, planName string,
 	}
 	return &searchResults.Items[0], nil
 }
+
+// CreatePlan creates a new plan.
+func (sdk *SDK) CreatePlan(opts CreatePlanFromOptions,
+) (Plan, error) {
+	fromPlan, err := sdk.RetrievePlanByName(opts.Name, ScopeOptions{
+		Scope:     opts.Scope,
+		Namespace: opts.Namespace,
+	})
+	if err != nil {
+		return nil, err
+	}
+	if opts.Scope.Matches(NamespaceScope) {
+		var plan = fromPlan.(*v1beta1.ServicePlan)
+		plan.Name = opts.Name
+		plan.Namespace = opts.Namespace
+		return sdk.createServicePlan(plan)
+	}
+	var plan = fromPlan.(*v1beta1.ClusterServicePlan)
+	plan.Name = opts.Name
+	return sdk.createClusterServicePlan(plan)
+}
+
+func (sdk *SDK) createClusterServicePlan(fromPlan *v1beta1.ClusterServicePlan,
```

Comment:
At this point I don't believe `fromPlan` is an accurate name for this argument since we have already copied from the original plan and modified the name. I suggest calling it `plan` instead.